src/components/countries/country/Country.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./country.scss";
+
+export const Country = ({ code, name, capital, continent }) => {
+  return (
+    <div className="country">

```

Comment:
Let's break down this component into smaller parts, for example
```suggestion
    <div className="country">
         <CountryFlag code={code} name={name} />
         <CountryCardContent {...contentProps}/>
    </div>
```
Please keep in mind that you will need country flag in other components in the future, so try to make it flexible, for example, add property that will control the size of the image.

Also, I would rename this component `Country` -> `CountryCard` since it would represent better what this compoent does and how it looks like.